### DISCUSSION TITLE
Usage of black code formatter

### DISCUSSION THREAD

**Discussion by coder2020official** [eternnoir/pyTelegramBotAPI#1866] [Ideas]
I'd recommend using [black code formatter](https://github.com/psf/black) in the project.

**What problem does it solve?**
It solves pep 8 issues in code. Since library has a lot of pep 8 issues, I think it'd be better to integrate it into our project.

**How does it work?**
I can install it, apply changes to our project, then make a PR. 

**Background**
I've already recommended using it, but by that time it wasn't stable and was in beta testing. However, currently, it is stable and it is used by many projects:
<img width="866" alt="image" src="https://user-images.githubusercontent.com/63740404/210004124-0322b932-8732-4397-be22-7f4caecb5ea4.png">

Thoughts, @Badiboy ?

**Comment by coder2020official** [comment]
For more:
https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html

**Comment by coder2020official** [comment]
We have tests in the repo, so if there're some issues in syntax, tests are gonna fail. So it is backward compatible.

**Comment by Badiboy** [comment]
> Thoughts, @Badiboy ?

My thoughts did not changed since the last time you propose it. I don't see it brings something valuable, so it will be just the waste of time.

However if you want to have fun bringing things that are valueless to my oppininon - do it. You are the active supporter of the library, you have enough rights to do it.

**Comment by coder2020official** [comment]
It is not valueless. We have to keep project clean. By the way, it is not me who cleans whole code myself - it is an automated system, so it is not waste of time.
